homeassistant/components/airvisual_pro/sensor.py

```diff
@@ -0,0 +1,165 @@
+"""Support for AirVisual Pro sensors."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_MILLION,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from . import AirVisualProEntity
+from .const import DOMAIN
+
+SENSOR_KIND_AQI = "air_quality_index"
+SENSOR_KIND_BATTERY_LEVEL = "battery_level"
+SENSOR_KIND_CO2 = "carbon_dioxide"
+SENSOR_KIND_HUMIDITY = "humidity"
+SENSOR_KIND_PM_0_1 = "particulate_matter_0_1"
+SENSOR_KIND_PM_1_0 = "particulate_matter_1_0"
+SENSOR_KIND_PM_2_5 = "particulate_matter_2_5"
+SENSOR_KIND_SENSOR_LIFE = "sensor_life"
+SENSOR_KIND_TEMPERATURE = "temperature"
+SENSOR_KIND_VOC = "voc"
+
+SENSOR_DESCRIPTIONS = (
+    SensorEntityDescription(
+        key=SENSOR_KIND_AQI,
+        name="Air quality index",
+        device_class=SensorDeviceClass.AQI,
+        native_unit_of_measurement="AQI",
```

Comment:
This device class should not have a unit of measurement. Either remove the unit of measurement or the device class.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes